201-decrypt-running-linux-vm/updateEncryptionSettings.json

```diff
@@ -0,0 +1,29 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "vmName": {
+            "type": "string",
+            "metadata": {
+                "description": "Name of the Virtual Machine"
+            }
+        }
+    },
+    "resources": [
+        {
+            "apiVersion": "2016-03-30",
```

Comment:
The latest apiVersion for virtual machine resources is "2015-06-15". Please use that value here, and then fix the schema validation errors.